neo/Prompt/Commands/Send.py

```diff
@@ -231,7 +231,7 @@ def parse_and_sign(wallet, jsn):
         context = ContractParametersContext.FromJson(jsn)
         if context is None:
             print("Failed to parse JSON")
-            return
+            return False
```

Comment:
It is a bad practice to mix return types. Now we get `boolean` or `Transaction`.  I know this is already happening throughout the code, but let's not add more as it will lead to maintainability problems. You'll see that the general style used is: 
```
x = create_object()
if x :
   # do something
else:
   # could not create object
```
`None` will also test for `False` in your test cases so that shouldn't be a problem